### PR TITLE
Feature/delete member from channel async tests

### PR DESCRIPTION
**Pull request by Gredja**
Created tests

### CODE REVIEW

File: Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+            // Assert
+            act.Should().Throw<NetKitChatNotFoundException>()
+                .And.Message.Should().Be($"Unable to delete member from channel. Member {nameof(saasUserId)}:{saasUserId} not found.");
+
+            VerifyMocks();
+        }
+
+        [Fact]
+        public void ShouldThrowIfMembeкIdEqualsMemberToDeleteId()
```
Comment (eugenekirvel) [inline Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs]: typo

File: Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+            // Assert
+            act.Should().Throw<NetKitChatNotFoundException>()
+                .And.Message.Should().Be($"Unable to delete member from channel. Channel {nameof(channelId)}:{channelId} not found.");
+
+            VerifyMocks();
+        }
+
+        [Fact]
+        public void ShouldThrowIfMembeкIdEqualChannelCreatorId()
```
Comment (eugenekirvel) [inline Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs]: typo

Comment (eugenekirvel) [reply Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs]: ShouldThrowIfMemberIdDoesNotEqualToChannelCreatorId

File: Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+            // Assert
+            act.Should().Throw<NetKitChatInvalidOperationException>()
+                .And.Message.Should().Be($"Unable to delete member from channel. Member {nameof(memberToDeleteId)}:{memberToDeleteId} is not joined to channel channelId:{channel.Id}.");
+
+            VerifyMocks();
+        }
+
+        [Fact]
+        public async Task ShouldReturnTaskCompleted()
```
Comment (eugenekirvel) [inline Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs]: check that message successfully deleted

File: Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs (hunk 1 above)
Comment (Gredja) [reply Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs]: Done

File: Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs (hunk 2 above)
Comment (Gredja) [reply Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs]: Renamed

File: Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs (hunk 3 above)
Comment (Gredja) [reply Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs]: Done

File: Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+            _channelRepositoryMock.Setup(x => x.GetChannelAsync(It.IsAny<Guid>()))
+                .ReturnsAsync((Channel)null)
+                .Verifiable();
+
+            // Act
+            Func<Task> act = async () => { await _channelService.DeleteMemberFromChannelAsync(member.SaasUserId, channelId, memberToDeleteId); };
+
+            // Assert
+            act.Should().Throw<NetKitChatNotFoundException>()
+                .And.Message.Should().Be($"Unable to delete member from channel. Channel {nameof(channelId)}:{channelId} not found.");
```
Comment (eugenekirvel) [inline Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs]: Channel {nameof(channelId)}:{channelId} is not found.

File: Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+            _memberRepositoryMock.Setup(x => x.GetMemberBySaasUserIdAsync(It.IsAny<string>()))
+                .ReturnsAsync((Member)null)
+                .Verifiable();
+
+            // Act
+            Func<Task> act = async () => { await _channelService.DeleteMemberFromChannelAsync(saasUserId, channelId, memberToDeleteId); };
+
+            // Assert
+            act.Should().Throw<NetKitChatNotFoundException>()
+                .And.Message.Should().Be($"Unable to delete member from channel. Member {nameof(saasUserId)}:{saasUserId} not found.");
```
Comment (eugenekirvel) [inline Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/DeleteMemberFromChannelAsyncTests.cs]: is not found

**Pull request merged**